starter-code/src/functions-and-arrays.js

```diff
@@ -33,6 +90,26 @@ var wordsArr = [
   'palace'
 ];
 
+function averageWordLength (wordList) {
+  var wordLengths = [];
+  var sum = 0;
+
+  if (wordList.length === 0) {
+    return undefined;
+  } else { 
+    for (var i = 0; i < wordList.length; i++) {
+      wordLengths.push(wordList[i].length);
+    };
+
+    for (var j = 0; j < wordLengths.length; j++) {
+      sum = sum + wordLengths[j];
+    };
+
+    var avg = sum / wordLengths.length;
```

Comment:
You could have used the averageNumbers function that you used above to calculate the average here